public/js/event-app.js

```diff
@@ -182,8 +182,33 @@ $(document).ready(function() {
         logger.log("Initialized app.");
 
         $("#admin-page-for-event").attr("href", "/admin/event/" + curEvent.id);
+        
+        var participantIndex = 1; 
+
+        $("#superuser-page-for-followupemail").attr("href", "/followup/event/" 
+            + curEvent.id + "/participant_" + participantIndex);
+        
         $(".about-event-container").hide();
 
+        //Show contact form on event page if either 
+        //preferred contact is null or noShare is false 
+
+        if(preferredContact == false && (noShare == false || noShare == undefined)) {
```

Comment:
Comment says ``null`` and code says ``false`` -- which is right?  Remember in js, ``false == 0``, ``undefined == null``, but ``false != null`` and ``false != undefined``.  Which is why ``===`` is usually preferred. :)